src/AppBundle/Twig/SourceCodeExtension.php

```diff
@@ -95,7 +95,7 @@ private function getTemplateCode()
 
     private function getTemplateRelativePath()
     {
-        return 'app/Resources/views/'.$this->template->getTemplateName();
+        return 'app'.DIRECTORY_SEPARATOR.'Resources'.DIRECTORY_SEPARATOR.'views'.DIRECTORY_SEPARATOR.$this->template->getTemplateName();
```

Comment:
I suggest adding a comment in the method phpdoc warning about this though, in case people read the code of the demo and want to do the same in their own place